extensions/amp-analytics/0.1/vendors.js

```diff
@@ -84,6 +84,51 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'burt': {
+    'vars': {
+      'trackingKey': 'ignore',
+      'category': '',
+      'subCategory': '',
+    },
+    'requests': {
+      'host': '//${trackingKey}.c.richmetrics.com/',
+      'base': '${host}imglog?e=${trackingKey}&pi=${trackingKey}|${pageViewId}|${canonicalPath}|${clientId(burt-amp-user-id)}&ui=${clientId(burt-amp-user-id)}&v=amp&ts=${timestamp}&sn=${requestCount}&',
+      'pageview': '${base}' +
+        'type=page&' +
+        'ca=${category}&' +
+        'sc=${subCategory}&' +
+        'ln=${browserLanguage}&' +
+        'lr=${documentReferrer}&' +
+        'eu=${sourceUrl}&' +
+        'tz=${timezone}&' +
+        'pd=${scrollWidth}x${scrollHeight}&' +
+        'sd=${screenWidth}x${screenHeight}&' +
+        'wd=${availableScreenWidth}x${availableScreenHeight}&' +
+        'ws=${scrollLeft}x${scrollTop}',
+      'pageping': '${base}' +
+        'type=pageping',
+    },
+    'triggers': {
```

Comment:
dong this will result in you guys getting a hit as soon as someone adds a amp-analytics tag with type == burt. I would recommend moving the pageview trigger to example and explicitly asking the publishers to include that in configuration but I'll leave the decision of doing that to you.